pkg/noun/v2/nock.h

```diff
@@ -0,0 +1,21 @@
+/// @file
+
+#ifndef U3_NOCK_V2_H
+#define U3_NOCK_V2_H
+
+#include "pkg/noun/nock.h"
+
+#include "types.h"
+
+  /**  Aliases.
+  **/
+#     define  u3n_v2_memo     u3n_memo
+#     define  u3n_v2_prog     u3n_prog
+#     define  u3n_v2_reclaim  u3n_reclaim
```

Comment:
This also needs to fix up pointers for portability. Also, these v2 aliases should point to v3.